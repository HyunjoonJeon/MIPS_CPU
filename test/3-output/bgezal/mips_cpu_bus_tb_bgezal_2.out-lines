register_v0:bfc0001c
