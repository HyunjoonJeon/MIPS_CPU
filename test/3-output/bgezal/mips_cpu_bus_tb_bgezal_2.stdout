WARNING: rtl/avl_slave_mem.v:38: $readmemh: Standard inconsistency, following 1364-2005.
WARNING: rtl/avl_slave_mem.v:38: $readmemh(test/1-binary/bgezal/bgezal_2.hex.txt): Not enough words in the file for the requested range [0:2047].
VCD info: dumpfile test/mips_cpu_bus.vcd opened for output.

TB : finished; active=0
register_v0:bfc0001c
